test/e2e/elasticsearch_test.go

```diff
@@ -80,22 +81,38 @@ func (suite *ElasticSearchTestSuite) TestSparkDependenciesES() {
 }
 
 func (suite *ElasticSearchTestSuite) TestSimpleProd() {
+	jaegerInstanceName := "simple-prod"
+	runSimpleProdTest(jaegerInstanceName, false)
+}
+
+func (suite *ElasticSearchTestSuite) TestSimpleProdWithOTEL() {
+	if !testOtelCollector {
+		t.Skip("Skipping OTEL collecotor test")
+	}
+	jaegerInstanceName := "simple-prod-with-otel"
+	runSimpleProdTest(jaegerInstanceName, true)
+}
+
+func runSimpleProdTest(name string, useOtelCollector bool) {
```

Comment:
Why is name being parametrized? How does it help with debugging?

Reply:
It's parameterized so that I can use a different instance name when running with or without the otel collector.   I can't remember details off the top of my head but we've had times where reusing the same instance name has caused problems.